examples/MQ/serialization/data/MyDigi.h

```diff
@@ -32,9 +32,21 @@ class access;
 class MyDigi : public FairTimeStamp
 {
   public:
-    MyDigi();
-    MyDigi(Int_t x, Int_t y, Int_t z, Double_t timeStamp, Double_t timeStampError = 0.0);
-    ~MyDigi() override;
+    MyDigi()
+        : FairTimeStamp()
+        , fX(0)
+        , fY(0)
+        , fZ(0)
```

Comment:
Could you move those into default member initializers?

With those in place, this constructor should be able to be `= default;` as well?

Reply:
Done.